### ISSUE TITLE
save results / download link

### ISSUE THREAD

**Issue by hitz** [hitz/RegulomeDB#9]
Need to store results as a file on server and provide download link.  Need final definition of file format(s)

How long should results remain on server?   Should be done when "big files" are done

**Comment by euriehong** [comment]
File formats for download should be GFF3 and BED at minimum.  

Also need to be able to link over to view hits in ENSEMBL and UCSC genome browser.  Syntax for linking:

UCSC genome browser (documentation: http://genome.ucsc.edu/goldenPath/help/customTrack.html#share)

http://genome.ucsc.edu/cgi-bin/hgTracks?org=human&position=chrX:55024742-55054742&hgt.customText=http://fafner.stanford.edu/~eurie/ALAS2-20110913-references.bed

ENSEMBL:

http://www.ensembl.org/Homo_sapiens/Location/View?r=X:55024742-55054742;contigviewbottom=url:http://fafner.stanford.edu/~eurie/ALAS2-20110913-references.bed

We can use any genomic region to initially construct the URL.  We can just use the position of the top scoring SNP

**Comment by hitz** [comment]
downloading in BED, GFF and "full" (all results) format.  Format slightly changed from orginal with _ delimiterss.
"Full" data is just ... preposterous for high scoring snps.  Many hundreds of : ... need to discuss this format.

Linking has been done for some time now. I mention that "data will remain on server for 48 hours" but there is currently no clean up script!

**Comment by hitz** [comment]
OK, I changed the "full" download format to something like what Alan did for the download files -

Differences are that:
1) I added 2nd field "Annotation" to "Related_Data" - I thought this was useful and only 1 extra field
2) I got rid of all the cell type notations, so that DNAase-Seq etc. will only be reported once no matter how many cell types the expt. was repeated in.

I can change these back if you like.
I would still like the file name on download to be named something other than "download" but I havn't figured out how to set the content-disposition correctly...
